test/make.test.ts

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs';
+import path from 'path';
+import dir from 'node-dir';
+import {createProject} from '../src/make';
+import {Contract, Frontend, TestingFramework} from '../src/types';
+
+describe('create', () => {
+  const testMatrix = [
+    ['js', 'react', 'js'],
+    ['js', 'vanilla', 'js'],
+    ['js', 'none', 'js'],
+    ['js', 'react', 'rust'],
+    ['js', 'vanilla', 'rust'],
+    ['js', 'none', 'rust'],
+    ['assemblyscript', 'react', 'js'],
+    ['assemblyscript', 'vanilla', 'js'],
+    ['assemblyscript', 'none', 'js'],
+    ['assemblyscript', 'react', 'rust'],
+    ['assemblyscript', 'vanilla', 'rust'],
+    ['assemblyscript', 'none', 'rust'],
+    ['rust', 'react', 'js'],
+    ['rust', 'react', 'rust'],
+    ['rust', 'vanilla', 'js'],
+    ['rust', 'vanilla', 'rust'],
+    ['rust', 'none', 'js'],
+    ['rust', 'none', 'rust'],
```

Comment:
This looks like it will take some effort to maintain. Can't there be elsewhere a definition of supported SC & FE languages and create a matrix programmatically from that?

Reply:
Agreed, fixed.